packages/font/tsconfig.build.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["src/**/*.test.ts"]
```

Comment:
If you add `exclude`, you should also exclude `node_modules`. That's the default in TypeScript but a specified `exclude` doesn't get merged, only overwritten. It doesn't necessarily matter here because no nested `node_modules` but it's a good habit to build

Reply:
My understanding is that this would only be needed if `node_modules` were included via `include` (or `files`). So this is needed in the default case because the default for `include` is `['**/*']`, but it shouldn't be needed here because `include` is `["src/**/*.ts"]` ... And I now learned that `include` is not inherited, so I will add it to `tsconfig.build.json` as well.